README.md

```diff
@@ -276,6 +276,9 @@ Router.map(function() {
 The route to transition to if the session is not authenticated can also be
 [configured](#configuration) to be another one than `login`.
 
+This mixin defines a method called `triggerAuthentication` that handles the routing to the configured authentication route.
+This method can be overridden if there is some custom behavior you desire.
+
```

Comment:
I think this is too much detail for the README actually. Having this method in the API docs should be fine.